CMakeLists.txt

```diff
@@ -105,8 +105,8 @@ endif()
 list(APPEND LCOV_EXCLUDE '${PROJECT_SOURCE_DIR}/perceptualdiff/*')
 
 # OSPRay rendering engine
-common_find_package(ospray 1.5 SYSTEM)
-set(_ospray_unsupported_version 1.6)
+common_find_package(ospray 1.7 SYSTEM)
+set(_ospray_unsupported_version 1.8)
```

Comment:
this doesn't make sense as we don't know yet if it might work with 1.8 or not. Just remove the usage of that `_ospray_unsupported_version`

Reply:
I'll remove it, but experience says that it won't work.

Reply:
Indeed